### PR TITLE
Create fr_fr.json

### PR DESCRIPTION
**Pull request by Calvineries**

### CODE REVIEW

**Review by xTracr — APPROVED** [review]

**Comment by xTracr** [comment]
Oh my god, I probably have found some issues in my second review: the translation of `pitch` and `yaw` might be reversed.  
I'm not quite sure about this because I just use machine translation to translate it back into English to review.  
If there are indeed some problems, you can directly reply to me how to change it, I will change the file in my next commit.

**Comment by Calvineries** [comment]
> Oh my god, I probably have found some issues in my second review: the translation of `pitch` and `yaw` might be reversed. I'm not quite sure about this because I just use machine translation to translate it back into English to review. If there are indeed some problems, you can directly reply to me how to change it, I will change the file in my next commit.

yes you are right, its:
```
"config.option.xtracr_realcamera_pitch": "Tangage",
"config.option.xtracr_realcamera_yaw": "Lacet",
"config.option.xtracr_realcamera_roll": "Roulis",
```
sorry

**Pull request merged**
